subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/slack/LintSlackReportTask.kt

```diff
@@ -39,7 +42,8 @@ abstract class LintSlackReportTask : DefaultTask() {
 
         createLintSlackAlert().report(
             lintReport = models,
-            channel = SlackChannel(slackReportChannel.get()),
+            channelForDevs = SlackChannel(slackReportChannel.get()),
```

Comment:
`ForDevs` - is a use case.
It's better to avoid such details in the plugin and focus on the content.

Reply:
i agree, fixed